### PR TITLE
Add android general plugin

### PR DESCRIPTION
**Pull request by snooky23**
## Description
Add a general plugin implementation for Android for URL scheme and Hook support.

## Checklist

* [X] PR is scoped to one task
* [ ] Tests are included
* [X] Documentation is included

* [X] This PR is not making any UI change
* [ ] This PR is making a UI change
  * [ ] Screenshot(s) included

## QA :

* required test cases:
* specific apps / plugins to test :

### CODE REVIEW

File: GeneralPlugin/Android/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# General Plugin example - Android
+
+This project serves as a base example on implementing a general plugin for iOS - with focus on using URL schemes and application lifecycle in order to launch activities.
```
Comment (liviur) [inline GeneralPlugin/Android/README.md]: you have the word iOS here

File: GeneralPlugin/Android/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+* `PluginSchemeI` - Implement this interface to handle calls for the Zapp URL scheme.
+* `ApplicationLoaderHookUpI` - Implement this interface in case you need to add soemting in the applicastion lifecycle.
+
+## Test your implementation with a development project
+
+* Download a development project from Zapp
+* Go to app/assets/applicaster.properties and set/add: `avoidRemotePluginConfigurationsFetching=true`.
+* Go to app/res/raw/plugin_configurations.json and add this JSON object to the bottom on the JSON array: 
+
+    ```
```
Comment (liviur) [inline GeneralPlugin/Android/README.md:16]: I like this

File: GeneralPlugin/Android/README.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    { 
+        "plugin": {
+            "api": {
+                "require_startup_execution": true,
+                "class_name": "com.funtoad.FuntoadAdapter"
+            },
+            "author_name": "developer eMail",
```
Comment (liviur) [inline GeneralPlugin/Android/README.md]: E-mail

File: GeneralPlugin/Android/plugin-manifest.json [hunk 4]
```diff
@@ -0,0 +1,33 @@
+{
+    "api": {
+        "require_startup_execution": false,
+        "class_name": "com.applicaster.web.plugins.iai.IAIWebAdapter"
```
Comment (liviur) [inline GeneralPlugin/Android/plugin-manifest.json]: `iai` is from a specific existing plugin right?
generalize this please

**Review by liviur — DISMISSED** [review]
added a few comments

File: GeneralPlugin/Android/README.md (hunk 1 above)
Comment (snooky23) [reply GeneralPlugin/Android/README.md]: Done

File: GeneralPlugin/Android/README.md (hunk 3 above)
Comment (snooky23) [reply GeneralPlugin/Android/README.md]: Done

File: GeneralPlugin/Android/plugin-manifest.json (hunk 4 above)
Comment (snooky23) [reply GeneralPlugin/Android/plugin-manifest.json]: I have removed it

**Review by lomudi — APPROVED** [review]
💯

File: GeneralPlugin/Android/README.md (hunk 2 above)
Comment (snooky23) [reply GeneralPlugin/Android/README.md:16]: 10x :)

**Pull request merged**